src/open_inwoner/templates/pages/kvk/branches.html

```diff
@@ -1,75 +1,50 @@
-                                    <h2 class="choice-list__heading">{{ entire_company.naam }}</h2>
-                                </div>
-                            </label>
-                        </li>
-                    {% endwith %}
-
-                    {% if company_branches|slice:"1:" %}
-                        <p><strong>{% trans "Of selecteer de vestiging waarmee u wilt inloggen" %}</strong></p>
-                        {% for branch in company_branches|slice:"1:" %}
-                            {% with company_id=branch.vestigingsnummer %}
-                                <li class="choice-list__item">
-                                    <label class="choice-list__label">
-                                        <input type="radio"
-                                               id="branch-{{ company_id }}"
-                                               name="branch_number"
-                                               value="{{ company_id }}"
-                                               class="choice-list__radio"
-                                               {% if user.vestiging == company_id %}checked{% endif %}>
-                                        <div class="choice-list__content">
-                                            <h2 class="choice-list__heading">{{ branch.naam }}</h2>
-                                            {% if company_id %}
-                                                <p class="choice-list__p">{% trans "Vestiging" %}: {{ company_id }}{% if branch.type == "hoofdvestiging" %} ({% trans "Hoofdvestiging" %}){% endif %}</p>
-                                            {% endif %}
-                                            {% if branch.adres.binnenlandsAdres.straatnaam %}
-                                                <p class="choice-list__p">{{ branch.adres.binnenlandsAdres.straatnaam }}</p>
-                                            {% endif %}
-                                            {% if branch.adres.binnenlandsAdres.plaats %}
-                                                <p class="choice-list__p">{{ branch.adres.binnenlandsAdres.plaats }}</p>
-                                            {% endif %}
-                                        </div>
-                                    </label>
-                                </li>
-                            {% endwith %}
-                        {% endfor %}
-                    {% endif %}
-
-                </ul>
+                {# React combobox with data only - translations handled by react-intl #}
+                <div
+                    data-react-module="selectcombobox"
+                    data-id="branch-selector"
+                    data-label="{% trans 'Selecteer de vestiging waarmee u wilt inloggen' %}"
+                    data-name="branch_number"
+                >
+                    <script type="application/json" class="branch-data">{% react_combobox_data company_branches user.vestiging %}</script>
```

Comment:
Would `id` make more sense? Is there a use-case for multiple instances?
```suggestion
                    <script type="application/json" id="branch-data">{% react_combobox_data company_branches user.vestiging %}</script>
```

Reply:
Not if we would rename it to something like SelectComboBoxVestigingen, but we may in future have multiple comboboxes in zaak filtering etc. So for this 1 component ID would be fine, but not for a future generic component, because "you never know...".